src/ensembl/src/content/app/browser/browser-nav/BrowserNavBar.scss

```diff
@@ -12,4 +14,49 @@
   &.browserNavBarExpanded {
     width: 100vw;
   }
+
+  dd {
+    margin-left: 10px;
+  }
```

Comment:
Is it a good time to gradually be removing `dd`s and `dl`s, at least from new / edited files? They are probably fine to be used for definitions (dl — definition list, dt — definition term, dd — definition description); but why do we keep using them for some other purposes?

Reply:
Should we be instead using `ul` then?

Reply:
Is it a list though?

Reply:
Yes, it is a list of icons.